docs/08-developer-guide/02-run/02-docker-compose.md

```diff
@@ -10,3 +10,143 @@ sidebar_label: Docker Compose
 - Define the Ozone runner image at `ozone-docker-runner` and its purpose to wrap Ozone binaries.
 - How to define which image gets used in the cluster (runner or other pre-built image).
 - Changing configurations in Docker Compose (`docker-config` file, transformation.py, etc).
+
+This guide explains how to run Apache Ozone using Docker Compose, either with locally built sources or pre-built images.
+
+## Prerequisites
+
+- Docker Engine 20.10.0 or higher
+- Docker Compose V2
+- Built Ozone distribution (if running from local build)
+
+## Running Ozone
+
+You can run Ozone either using your locally built version or using official pre-built Docker images.
```

Comment:
I think we should organize this a little differently. We can have one section on running docker-compose from a pre-build Ozone tarball, regardless of where the tarball came from (built from source or Ozone release). Then we can mention the pre-built docker images for each release separately if people want to use those for other things.

I don't see why someone would pull just the compose file from the tarball and manually pair it with a docker image built from the same version.